recipes/autoconf/all/test_package/conanfile.py

```diff
@@ -3,13 +3,20 @@
 import os
 import shutil
 
+required_conan_version = ">=1.36.0"
+
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     exports_sources = "configure.ac", "config.h.in", "Makefile.in", "test_package_c.c", "test_package_cpp.cpp",
+    test_type = "build_requires"
```

Comment:
I didn't know this attribute. Does it automatically add the recipe in build_requires instead of requires?

Reply:
https://docs.conan.io/en/latest/devtools/build_requires.html#testing-build-requires

Reply:
cci has some issues with it.
See the conversation at https://github.com/conan-io/conan-center-index/pull/6694#issuecomment-894134208